<?xml version="1.0"?>
<robot xmlns:xacro="http://ros.org/wiki/xacro" name="wam-v-thrusters">
  <xacro:include filename="$(find wamv_description)/urdf/thrusters/engine.xacro" />
    <!-- === engine === -->
      <xacro:engine prefix="front_left" position="1.1 1 0" orientation="0.0 0.0 -0.7854" />
      <xacro:engine prefix="front_right" position="1.1 -1 0" orientation="0.0 0.0 0.7854" />
      <xacro:engine prefix="back_left" position="-2.4 1 0" orientation="0.0 0.0 0.7854" />
      <xacro:engine prefix="back_right" position="-2.4 -1 0" orientation="0.0 0.0 -0.7854" />

      <xacro:include filename="$(find wamv_gazebo)/urdf/thruster_layouts/wamv_gazebo_thruster_config.xacro" />
        <xacro:wamv_gazebo_thruster_config name="front_left" />
        <xacro:wamv_gazebo_thruster_config name="front_right" />
        <xacro:wamv_gazebo_thruster_config name="back_left" />
        <xacro:wamv_gazebo_thruster_config name="back_right" />
</robot>